hack/prepare-supervisor-on-kind.sh

```diff
@@ -348,9 +428,20 @@ kubectl apply --namespace "$PINNIPED_TEST_SUPERVISOR_NAMESPACE" -f "$fd_file"
 echo "Waiting for FederationDomain to initialize or update..."
 kubectl wait --for=condition=Ready FederationDomain/my-federation-domain -n "$PINNIPED_TEST_SUPERVISOR_NAMESPACE"
 
+# Decide if we need to use the proxy settings for certain commands below.
+if [[ "${PINNIPED_USE_CONTOUR:-}" != "" ]]; then
+  proxy_server="$PINNIPED_TEST_PROXY"
+  proxy_except="127.0.0.1"
+  proxy_env_vars=""
+else
+  proxy_server="$PINNIPED_TEST_PROXY"
```

Comment:
The first two variables seem to get the same settings either way?

Reply:
Oh shoot! Good catch! Updated!

Reply:
latest change 👍